android/keymanager/src/main/assets/config.properties

```diff
@@ -17,9 +17,9 @@ mosip.kernel.crypto.asymmetric-algorithm-message-digest-function=SHA-256
 #Crypto asymmetric algorithm OAEP mask generation function
 mosip.kernel.crypto.asymmetric-algorithm-mask-generation-function=MGF1
 #Crypto symmetric IV length
-mosip.kernel.crypto.symmetric-algorithm-iv-length=12
+mosip.kernel.crypto.symmetric-algorithm-iv-length=16
```

Comment:
any specific reason for property change here

Reply:
That was a quick fix I got from Piyush for login. Not sure about the details. No specific reason.